src/routes/+page.svelte

```diff
@@ -29,9 +29,12 @@
 <svelte:head>
   <title>Svelte Promise Modals</title>
   <meta property="og:title" content="Svelte Promise Modals" />
-  <meta property="og:site_name" content="Svelte Promise Modals" />
+  <meta property="og:site_name" content="Svelte Promise Modals - Modals made easy." />
```

Comment:
Wait, the site name was not OK with LI?

Reply:
The one tool said this:  og:title metatag found
 Your title should be between 30-60 characters, with a maximum of 90 (currently 21 characters)